pkg/kube/inject/testdata/inject/user-volume.yaml.injected

```diff
@@ -192,8 +192,6 @@ spec:
           runAsGroup: 0
           runAsNonRoot: false
           runAsUser: 0
-      securityContext:
-        fsGroup: 1337
```

Comment:
noticed that the scripts removed a lot of the securityContext sections in the testdata  I think the behavior wanted is that if fsGroup is set then it would stay what the user wanted instead of being over-written with 1337.  However was unclear if there is no fsGroup set do we want it to default to 1337 or not have a value?

Reply:
@AndreaM12345 : testdata was auto-generated when I ran makegen.. Even before this PR, I think the expectation was that we set fsGroup to 1337 only when the feature flag is enabled. I understand your point that by default it was having fsGroup set even in the absence of flag, but that is because the default was to enable the fsGroupInjection?